src/esockd.appup.src

```diff
@@ -1,7 +1,11 @@
 %%-*-: erlang -*-
 
-{"5.7.5",
- [{"5.7.4",
+{"5.7.6",
```

Comment:
it's a new feature, update to 5.8 ?

Reply:
There are already 5.8.x and 5.9.x, this pr is for 5.7.x